modules/kubernetes-addons/kuberay-operator/main.tf

```
@@ -0,0 +1,30 @@
+locals {
+  name      = "kuberay-operator"
+  namespace = try(var.helm_config.namespace, local.name)
+}
+
+resource "kubernetes_namespace_v1" "this" {
+  metadata {
+    name = local.namespace
+  }
+}
+
+module "helm_addon" {
+  source = "../helm-addon"
+
+  helm_config = merge(
+    {
+      name        = local.name
+      chart       = "${path.module}/kuberay-operator-config"
+      version     = "0.1.0"
+      namespace   = local.namespace
```

if you pass in the attribute output you can remove the `depends_on`

```suggestion
      namespace   = kubernetes_namespace_v1.this.metadata[0].name
```

---

fixed